docs/using-rucio-client.md

```diff
@@ -145,21 +111,21 @@ daemon is running on the Rucio server side, one can also grant Rucio a
 refresh token and specify the time for which Rucio should act on behalf
 of the user (in hours) using the `--refresh-lifetime` option:
 
-> **rucio -a=<rucio_account_name> -S=OIDC --oidc-scope="openid profile offline_access" --oidc-refresh-lifetime=24 -v whoami**
-
+```bash
+rucio -a=<rucio_account_name -S=OIDC --oidc-scope="openid profile offline_access" --oidc-refresh-lifetime=24 -v whoami
+```
 If Rucio Server is granted a user both valid access and refresh tokens,
 it is also possible to configure Rucio Client to ask Rucio Server for
 token refresh. Assuming user used one of the 3 CLI authentication
 methods above + requested offline_access in the scope, rucio.cfg file
 can be configured with the following parameters in the \[client\]
```

Comment:
\[client\] needs to be rewritten as client.

Reply:
Also, 

rucio -a=<rucio_account_name -S=OIDC --oidc-scope="openid profile offline_access" --oidc-refresh-lifetime=24 -v whoami

needs to read as

rucio -a=<rucio_account_name> -S=OIDC --oidc-scope="openid profile offline_access" --oidc-refresh-lifetime=24 -v whoami